app/models/similar_occupation_standards.rb

```diff
@@ -1,19 +1,29 @@
 class SimilarOccupationStandards
-  attr_reader :occupation_standard
+  attr_reader :occupation_standard, :debug
 
   RESULTS_SIZE = 5
   MINIMUM_SCORE = 0.2
 
-  def self.similar_to(occupation_standard)
-    new(occupation_standard).similar_to
+  def self.similar_to(occupation_standard, debug = false)
+    new(occupation_standard, debug).similar_to
   end
 
-  def initialize(occupation_standard)
+  def initialize(occupation_standard, debug = false)
     @occupation_standard = occupation_standard
+    @debug = debug
   end
 
   def similar_to
-    OccupationStandard.__elasticsearch__.search(query).records
+    response = OccupationStandard.__elasticsearch__.search(query)
+    if debug
```

Comment:
@littleforest is there a performance concern to conditionally print debugging output? I don't see an issue with this method always logging with Rails.logger.info and having this data available all the time. What are your thoughts?

Reply:
I feel like `Rails.logger.info` is not that helpful to me in debugging tests, since it's easier to use `puts` and see the output directly that having it go into the log. And I'd prefer to not have `puts` always on since it makes the test output noisy.

We are using the `bonsai` gem for the ES connection, and it actually takes care of the Elasticsearch.client connection setup. Ultimately I'd like to do the client setup manually, because then we can pass the logging flag directly in the Elasticsearch.client set up and trigger it that way.

I don't intend to keep this `if debug` code around forever, but since I was still going to be doing some more work on this class, I thought I would leave it for the short term.

Reply:
If it's something temporal, we can keep it. Thanks for the clarification!